automatic/apache-httpd/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,6 @@
+$configFile = Join-Path $env:chocolateyPackageFolder 'config.json'
```

Comment:
I'm guessing this should actually be a config.json file located in the tools directory, not the root package directory (this would also be the reason why appveyor fails).